x-pack/plugins/security_solution/common/endpoint/service/trusted_apps/validations.test.ts

```diff
@@ -0,0 +1,537 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isPathValid } from './validations';
+import { OperatingSystem, ConditionEntryField } from '../../types';
+
+describe('Validate Windows paths', () => {
+  it('should return TRUE when paths are exact', () => {
```

Comment:
Wow. lots of `expect`'s here in this one test case. Suggestion to perhaps break it out into multiple tests (`it()`) so that if any fail in the future, it will be easier to identify?

you could even have different `describe()`'s to capture each of the sub-groups of tests.

Reply:
done 06ab0d527213c342288528bdda35744a5d51b204. Writing grouped test also helped me to see that the windows regex needed a bit of fixing.